README.md

```diff
@@ -0,0 +1,5 @@
+# Databricks quick start demo
+
+
+* Setup your Databricks account https://databricks.com/try-databricks
+* Configure your Databricks CLI with your default profile https://docs.databricks.com/dev-tools/cli/index.html
```

Comment:
Is there nothing else you need to run this demo? :)

Reply:
Right, in order to set up your Databricks account, you also need an AWS or Azure account.
And you also need JQ installed on your command line. 

Thanks, I will add that to the readme ;)